app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrUpdatePetWash.java

```diff
@@ -55,7 +55,7 @@ public void shouldUpdateWashDate() throws WashAlreadyExistingException,
         trNewPetWash.setPet(linux);
         trNewPetWash.setWash(originalWash);
         trNewPetWash.execute();
-        originalWash.setWashDate(DateTime.Builder.build(2011, 5, 27, 11, 15, 1));
+        originalWash.setDateTime(DateTime.Builder.build(2011, 5, 27, 11, 15, 1));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['15' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5526227)